src/app/components/challenge/challengeevaluation/challengeevaluation.component.html

```diff
@@ -2,39 +2,51 @@
 	<div class="row">
 		<div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 ev-pad">
 			<div class="ev-md-container ev-card-panel">
-				<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10 content">
-					<span class="fw-semibold fs-18">
-						<strong class="fw-semibold">Evaluation Criteria</strong>
-					</span>
-					<div class="evaluation-details" id="challenge-evaluation"></div>
+				<div class="row">
+					<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
+						<span class="fw-semibold fs-18">
+							<strong class="fw-semibold">Evaluation Criteria</strong>
+						</span>
+					</div>
+					<div *ngIf="isChallengeHost" class="col-sm-1 col-xs-1">
+						<a class="pointer" (click)="editEvaluationCriteria()">
+							<i class="fa fa-pencil" aria-hidden="true">
+							</i>
+						</a>
+					</div>
+					<div *ngIf="isChallengeHost" class="col-sm-1 col-xs-1">
+						<a class="pointer" (click)="editEvaluationScript()">
+							<i class="fa fa-file" aria-hidden="true">
+							</i>
+						</a>
+					</div>
 				</div>
-				<div *ngIf="isChallengeHost" class="col-sm-1 col-xs-1">
-					<a class="pointer" (click)="editEvaluationCriteria()">
-						<i class="fa fa-pencil" aria-hidden="true">
-						</i>
-					</a>
-				</div>
-				<div *ngIf="isChallengeHost" class="col-sm-1 col-xs-1">
-					<a class="pointer" (click)="editEvaluationScript()">
-						<i class="fa fa-file" aria-hidden="true">
-						</i>
-					</a>
+				<div class="row">
+            		<div class="col-sm-12">
+						<div class="evaluation-details" id="challenge-evaluation"></div>
+					</div>
 				</div>
 			</div>
 		</div>
 		<div class="col-lg-6 col-md-6 col-sm-6 col-xs-12 tnc-pad">
 			<div class="ev-md-container ev-card-panel">
-				<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10 content">
-					<span class="fw-semibold fs-18">
-						<strong class="fw-semibold">Terms And Conditions</strong>
-					</span>
-					<div class="terms-and-conditions" id="challenge-tnc"></div>
+				<div class="row">
+					<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
+						<span class="fw-semibold fs-18">
+							<strong class="fw-semibold">Terms And Conditions</strong>
+						</span>
+					</div>
+					<div *ngIf="isChallengeHost" class="col-sm-2 col-xs-2">
+						<a class="pointer" (click)="editTermsAndConditions()">
```

Comment:
There seems un-necessary large right margin for the edit (pencil) icon.

Reply:
@lunayach done